onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -160,6 +160,9 @@ def create_backend_test(testname=None):
         if c2.supports_device('NGRAPH'):
             current_failing_tests = current_failing_tests + ('|^test_clip*',)
 
+        if c2.supports_device('OPENVINO_GPU_FP32'):
```

Comment:
>OPENVINO_GPU_FP32 [](start = 31, length = 17)

shouldn't we add "OPENVINO_CONFIG_GPU_FP16" as well?

Reply:
yes, add OPENVINO_GPU_FP16 as well.

Reply:
Yes i missed this. I will add this change